src/proxy_services/user_task_proxy_service/user_task_proxy_service.ts

```diff
@@ -0,0 +1,205 @@
+
+import {mapToUserTask} from '../mapper_functions';
+
+export class UserTaskProxyService implements APIs.IUserTaskConsumerApi {
+  private flowNodeInstanceService: Services.IFlowNodeInstanceService;
+  private notificationService: Services.INotificationService;
+
+  constructor(
+    flowNodeInstanceService: Services.IFlowNodeInstanceService,
+    notificationService: Services.INotificationService,
+  ) {
+    this.flowNodeInstanceService = flowNodeInstanceService;
+    this.notificationService = notificationService;
+  }
+
+  public async getUserTasksForProcessModel(
+    identity: IIdentity,
+    processModelId: string,
+    offset?: number,
+    limit?: number,
+  ): Promise<DataModels.UserTasks.UserTaskList> {
+    const result = await this.flowNodeInstanceService.query(
+      identity,
+      {processModelId: processModelId, flowNodeType: UserDomainDataModels.FlowNodeInstances.QueryableBpmnType.userTask},
+      offset,
+      limit,
+    );
+
+    return this.mapToUserTaskList(result);
+  }
+
+  public async getUserTasksForProcessInstance(
+    identity: IIdentity,
+    processInstanceId: string,
+    offset?: number,
+    limit?: number,
+  ): Promise<DataModels.UserTasks.UserTaskList> {
+    const result = await this.flowNodeInstanceService.query(
+      identity,
+      {
+        processInstanceId: processInstanceId,
+        flowNodeType: UserDomainDataModels.FlowNodeInstances.QueryableBpmnType.userTask,
+      },
+      offset,
+      limit,
+    );
+
+    return this.mapToUserTaskList(result);
+  }
+
+  public async getUserTasksForCorrelation(
+    identity: IIdentity,
+    correlationId: string,
+    offset?: number,
+    limit?: number,
+  ): Promise<DataModels.UserTasks.UserTaskList> {
+    const result = await this.flowNodeInstanceService.query(
+      identity,
+      {correlationId: correlationId, flowNodeType: UserDomainDataModels.FlowNodeInstances.QueryableBpmnType.userTask},
+      offset,
+      limit,
+    );
+
+    return this.mapToUserTaskList(result);
+  }
+
+  public async getUserTasksForProcessModelInCorrelation(
+    identity: IIdentity,
+    processModelId: string,
+    correlationId: string,
+    offset?: number,
+    limit?: number,
+  ): Promise<DataModels.UserTasks.UserTaskList> {
+    const result = await this.flowNodeInstanceService.query(
+      identity,
+      {
+        processModelId: processModelId,
+        correlationId: correlationId,
+        flowNodeType: UserDomainDataModels.FlowNodeInstances.QueryableBpmnType.userTask,
+      },
+      offset,
+      limit,
+    );
+
+    return this.mapToUserTaskList(result);
+  }
+
+  public async getWaitingUserTasksByIdentity(
+    identity: IIdentity,
+    offset?: number,
+    limit?: number,
+  ): Promise<DataModels.UserTasks.UserTaskList> {
+    const result = await this.flowNodeInstanceService.query(
+      identity,
+      {flowNodeType: UserDomainDataModels.FlowNodeInstances.QueryableBpmnType.userTask},
```

Comment:
s.o., damit sich das hier so wie in der PE verhält, muss hier noch ein matching auf `ownerId` gemacht werden.

```suggestion
      {
        flowNodeType: UserDomainDataModels.FlowNodeInstances.QueryableBpmnType.userTask,
        ownerId: identity.userId,
      },
```